core/storage/question/gae_models_test.py

```diff
@@ -251,6 +307,23 @@ def test_delete_multi_question_skill_link(self) -> None:
         self.assertEqual(len(question_skill_links), 1)
         self.assertEqual(question_skill_links[0].question_id, 'question_id3')
 
+    def test_cannot_link_same_question_to_given_skill(self) -> None:
+        question_skill_link_model = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id1', 'skill_id1', 0.1)
+            )
+
+        question_models.QuestionSkillLinkModel.put_multi_question_skill_links([
+            question_skill_link_model
+        ])
+
+        with self.assertRaisesRegex(  # type: ignore[no-untyped-call]
+            Exception,
+            'The given question is already linked to given skill'
```

Comment:
Could you please change the error message, here and in the non-test code, to include the question ID and the skill ID, i.e. "The question with ID XYZ is already linked to skill ABC"? Otherwise, I don't think this error message will be helpful if it occurs in practice.

Reply:
Done!